src/components/PaletteSelector.js

```diff
@@ -16,21 +18,53 @@ const PaletteSelector = () => {
     activePalette: ""
   };
 
+  const activeTab = {
+    fireTabIsActive: "color-palette-btn active fire-active",
+    leafTabIsActive: "color-palette-btn not-active",
+    waveTabIsActive: "color-palette-btn not-active",
+    moonTabIsActive: "color-palette-btn not-active"
+  }
+
     switch (event.target.id) {
       case 'fire-palette-btn':
         setActivePalette({...noPalettesActive, isFireActive: true})
+        setActiveTab({    
+          fireTabIsActive: "color-palette-btn active fire-active",
+          leafTabIsActive: "color-palette-btn not-active leaf-inactive",
+          waveTabIsActive: "color-palette-btn not-active wave-inactive",
+          moonTabIsActive: "color-palette-btn not-active moon-inactive"
+        })
+        
         break;
 
       case 'leaf-palette-btn':
         setActivePalette({...noPalettesActive, isLeafActive: true})
+        setActiveTab({    
+          fireTabIsActive: "color-palette-btn not-active fire-inactive",
+          leafTabIsActive: "color-palette-btn active leaf-active",
+          waveTabIsActive: "color-palette-btn not-active wave-inactive",
+          moonTabIsActive: "color-palette-btn not-active moon-inactive"
+        })
```

Comment:
Could you use the spread operator for these lines and those for the wave and moon tabs?